downstream/modules/eda/con-system-level-monitoring.adoc

```diff
@@ -0,0 +1,15 @@
+[id="system-level-monitoring"]
+
+=  System level monitoring for {EDAcontroller}
+
+[role="_abstract"]
+After characterizing your workload to determine how many rulebook activations you are running in parallel and how many events you are receiving at any given point, you must consider monitoring your {EDAcontroller}  host at the system level. Using system level monitoring to review information about {EDAName}’s performance over time helps when diagnosing problems or when capacity planning for future growth.
```

Comment:
```suggestion
After characterizing your workload to determine how many rulebook activations you are running in parallel and how many events you are receiving at any given point, you must consider monitoring your {EDAcontroller}  host at the system level. Using system level monitoring to review information about {EDAName}’s performance over time helps when diagnosing problems or when considering capacity for future growth.
```
Just a suggestion - this might be easier for auto-translate to parse